tests/integration/test_events.py

```diff
@@ -240,6 +240,39 @@ def test_put_events_with_target_sqs(self, events_client, sqs_client):
             events_client, sqs_client, pattern=TEST_EVENT_PATTERN, entries_asserts=[(entries, True)]
         )
 
+    @pytest.mark.aws_validated
```

Comment:
```suggestion
```

With this specific test setup, this unfortunately doesn't work against AWS right now, which is why I'm removing the marker here. This is more of an issue with `_put_events_with_filter_to_sqs` though. The event test cases themselves (the 3 entries) should be fine and worked as expected when I tested them in a different test setup.

I'll let this pass for now and will refactor the tests in here soon, so we can mark them all `aws_validated` :+1: